types/config.d.ts

```diff
@@ -64,6 +64,8 @@ declare global {
          * Ignore means a failure is expected and no action should be taken.
          */
         loadFailureMode?: 'fatal'|'warn'|'ignore';
+        /** Whether the pass should skip the page load. If false, gathering begins as soon as navigation completes. */
+        skipPageLoad?: boolean;
```

Comment:
```suggestion
        skipWaitingForPageLoad?: boolean;
```
just an idea and a lot longer, but `skipPageLoad` also kind of reads like "skip loading [navigating to] the page", so this might be a bit clearer?